### ISSUE TITLE
application is crashed on some data

### ISSUE THREAD

**Issue by tim2CF** [svetob/logstash-json#24]
```
GenEvent handler {LogstashJson.Console, :json} installed in Logger terminating
** (FunctionClauseError) no function clause matching in Poison.Encoder.BitString.chunk_size/3
    (poison) lib/poison/encoder.ex:139: Poison.Encoder.BitString.chunk_size(<<171, 152, 70, 16, 37>>, nil, 0)
    (poison) lib/poison/encoder.ex:134: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:95: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:101: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:95: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:88: Poison.Encoder.BitString.encode/2
    (poison) lib/poison/encoder.ex:227: anonymous fn/4 in Poison.Encoder.Map.encode/3
    (poison) lib/poison/encoder.ex:228: Poison.Encoder.Map."-encode/3-lists^foldl/2-0-"/3
```

**Comment by svetob** [comment]
Interesting, do you know which type of data is causing this, or could provide more info?

**Comment by tim2CF** [comment]
yes, 
response body was google protobuf binary message
full stacktrace was
```
GenEvent handler {LogstashJson.Console, :json} installed in Logger terminating
** (FunctionClauseError) no function clause matching in Poison.Encoder.BitString.chunk_size/3
    (poison) lib/poison/encoder.ex:139: Poison.Encoder.BitString.chunk_size(<<171, 152, 70, 16, 37>>, nil, 0)
    (poison) lib/poison/encoder.ex:134: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:95: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:101: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:95: Poison.Encoder.BitString.escape/2
    (poison) lib/poison/encoder.ex:88: Poison.Encoder.BitString.encode/2
    (poison) lib/poison/encoder.ex:227: anonymous fn/4 in Poison.Encoder.Map.encode/3
    (poison) lib/poison/encoder.ex:228: Poison.Encoder.Map."-encode/3-lists^foldl/2-0-"/3
Last message: {:info, #PID<0.383.0>, {Logger, "some-secret-string", {{2018, 6, 19}, {3, 39, 53, 873}}, [pid: #PID<0.26369.0>, application: :lab, module: Lab.Flask, function: "remote_call/3", file: "some-secret-string", line: 188, elapsed_time: 13, result: {:ok, %HTTPoison.Response{body: <<10, 6, 8, 171, 152, 70, 16, 37>>, headers: [{"server", "Cowboy"}, {"date", "Tue, 19 Jun 2018 03:39:53 GMT"}, {"content-length", "8"}, {"content-type", "application/octet-stream; charset=utf-8"}, {"cache-control", "max-age=0, private, must-revalidate"}, {"x-request-id", "some-secret-string"}], request_url: "some-secret-string", status_code: 200}}, request_url: "some-secret-string", substance_enum: :TRANSACTOR, act_enum: :GET_BALANCE, act_kind: :REACTION]}}
```

**Comment by svetob** [comment]
Fixed in #25
